packages/cms-lib/lib/config.js

```diff
@@ -131,24 +137,12 @@ const isConfigPathInGitRepo = () => {
   return configDir.startsWith(gitDir);
 };
 
-const CONFIG_FILE_WILDCARD_MATCHER = 'hubspot.config.*';
-
-const configFilenameIsIgnoredByGitignore = () => {
-  const ignoreFiles = getGitignoreFiles();
-  const configPathSegments = _configPath.split('/');
-  const configFilename = configPathSegments[configPathSegments.length - 1];
-
+const configFilenameIsIgnoredByGitignore = ignoreFiles => {
   return ignoreFiles.some(gitignore => {
-    const gitignoreContents = fs
-      .readFileSync(gitignore, 'utf8')
-      .split('\n')
-      .filter(i => i);
+    const gitignoreContents = fs.readFileSync(gitignore).toString();
     const gitignoreConfig = ignore().add(gitignoreContents);
-    if (
-      gitignoreConfig.ignores(configFilename) ||
-      gitignoreConfig.ignores(CONFIG_FILE_WILDCARD_MATCHER)
-    ) {
-      // Has a gitignore rule
+
+    if (gitignoreConfig.ignores(getConfigFilename())) {
```

Comment:
I think this needs to be a path relative to the location of the gitignore file being processed.